File: src/main/java/com/bajdcc/LALR1/interpret/os/task/TKUI.java
package com.bajdcc.LALR1.interpret.os.task;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【服务】用户界面
 *
 * @author bajdcc
 */
public class TKUI implements IOSCodePage {
	@Override
	public String getName() {
		return "/task/ui";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/java/com/bajdcc/LALR1/interpret/os/task/TKUI.kt
package com.bajdcc.LALR1.interpret.os.task

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【服务】用户界面
 *
 * @author bajdcc
 */
class TKUI : IOSCodePage {
    override val name: String
        get() = "/task/ui"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
